docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -427,23 +427,53 @@ the regular expression `/^nginx-[a-z0-9-]+$/`, run the following command:
 $ tsh request create --resources /teleport.example.com/pod/mycluster/*/^nginx-[a-z0-9-]+$
 ```
 
-<Notice type="warning">
+If a user has no access to a Kubernetes cluster, he can search the list of pods
+in the cluster by running the following command:
 
-Teleport does not currently support `tsh request search --kind=pods`.
+```code
+$ tsh request search --kind=pod --kube-cluster=<Var name="kube-cluster" /> [--kube-namespace=<Var name="namespace" />|--all-kube-namespaces]
+Name               Namespace Labels    Resource ID
+-----------------  --------- --------- ----------------------------------------------------------
+nginx-deployment-0 default   app=nginx /teleport.example.com/pod/local/default/nginx-deployment-0
+nginx-deployment-1 default   app=nginx /teleport.example.com/pod/local/default/nginx-deployment-1
 
-</Notice>
+To request access to these resources, run
+> tsh request create --resource /teleport.example.com/pod/local/default/nginx-deployment-0 --resource /teleport.example.com/pod/local/default/nginx-deployment-1 \
+    --reason <request reason>
+```
+
+The list returned includes the name of the pod, the namespace it is in, its labels
+and the resource ID. Pods included in the list are those that match the
+`kubernetes_resources` field in the user's `search_as_roles`. The user can then
```

Comment:
Let's check with @alexfornuto and @ptgott on this language. I don't know if we often refer to "the user" and "he" consistently.

Reply:
I made a copy edit commit addressing consistent and gender-neutral reference to the user, along with other adjustments.

Reply:
Thanks @alexfornuto